app/serializers/host_serializer.rb

```diff
@@ -18,4 +18,8 @@ class HostSerializer
     host.all_profiles
   end
   # rubocop:enable Style/SymbolProc
+
+  has_many :policies do |host|
+    host.assigned_profiles.external(false)
+  end
```

Comment:
So far, we haven't polluted the REST API with this concept that a policy is its initial profile, etc. I'd like to keep it that way. If this is returning profiles, it should be named appropriately. IMO it should be as explicit as possible, like `assigned_internal_profiles` or similar.

I'm curious why we're even changing the REST API - I expected this PR to be GraphQL only.

Reply:
I thought that the parity between APIs was needed. I can take the REST API changes out.

Reply:
REST API changes dropped.